### DISCUSSION TITLE
Create note from item in calendar view

### DISCUSSION THREAD

**Discussion by CaryJM** [ivan-lednev/obsidian-day-planner#588] [Q&A]
Hello - Great plugin, appreciate the work. Wondering if I'm missing something with how this plugin works and if anyone can help me out. I'd like to use the calendar block view to readily create new notes based on an item (a meeting), ideally by doing something like clicking on an item and automatically creating a new note based on the meeting name and date. I can imagine optional functionality, such as pulling in the names of meeting invitees etc but this is less critical. Any thoughts or recommendations?

Thanks!

**Comment by ivan-lednev** [comment]
Hello, this is not currently implemented, but it's on my roadmap.
